graphql/resolvers.go

```diff
@@ -20,14 +23,14 @@ func (r *Resolver) Query() QueryResolver {
 
 type patchResolver struct{ *Resolver }
 
-func (r *patchResolver) ID(ctx context.Context, obj *model.APIPatch) (string, error) {
+func (r *patchResolver) ID(ctx context.Context, obj *restModel.APIPatch) (string, error) {
 	return *obj.Id, nil
 }
 
 type queryResolver struct{ *Resolver }
 
-func (r *queryResolver) UserPatches(ctx context.Context, userID string) ([]*model.APIPatch, error) {
-	patchPointers := []*model.APIPatch{}
+func (r *queryResolver) UserPatches(ctx context.Context, userID string) ([]*restModel.APIPatch, error) {
```

Comment:
is there anyway to avoid returning slices of pointers? they're quite confusing to use as a caller, and in this kind of case don't really offer benefits.

Reply:
i don't think so :( it is a drawback of the gql-gen generated code which demands the return type of pointers for some reason